src/profile-logic/process-profile.js

```diff
@@ -223,26 +248,117 @@ function processThread(thread, libs) {
             );
           }
           fileName = stringTable.indexForString(scriptURI);
-          lineNumber = jsMatch[3] | 0;
+          lineNumber = parseInt(jsMatch[3], 10);
```

Comment:
I've been bit before by this in the past I feel... `parseInt("0x10")` is still `16`, so I don't mind being explicit with it.